crates/tako/src/tests/test_reactor.rs

```diff
@@ -1042,3 +1041,23 @@ fn cancel_tasks<T: Into<TaskId> + Copy>(core: &mut Core, task_ids: &[T]) {
         &task_ids.iter().map(|&v| v.into()).collect::<Vec<_>>(),
     );
 }
+
+#[test]
+fn test_task_deps() {
+    let mut core = Core::default();
+    //create_test_workers(&mut core, &[1, 1, 1]);
+    submit_example_3(&mut core);
+    assert_eq!(core.get_read_to_assign().len(), 2);
+    create_test_workers(&mut core, &[1]);
+    start_and_finish_on_worker(&mut core, 2, 100, 0);
+    assert!(core.get_task_by_id_or_panic(3.into()).get().is_waiting());
```

Comment:
Let's introduce helper methods like `assert_waiting([3, 4, 6])` and `assert_ready([5])`. It's PITA to change tests that have some method call repeated many times when the method changes.